src/osd/PrimaryLogPG.cc

```diff
@@ -4404,7 +4404,7 @@ struct C_ChecksumRead : public Context {
   OSDOp &osd_op;
   Checksummer::CSumType csum_type;
   bufferlist init_value_bl;
-  ceph_le64 read_length;
+  ceph_le64 read_length{};
```

Comment:
`ceph_le64` seems to be an integer. So better initialize with `0` to be specific.